doc/manual/io.rst

```diff
@@ -350,6 +350,15 @@ The `BDF format <http://www.biosemi.com/faq/file_format.htm>`_ is a 24-bit
 variant of the EDF format used by the EEG systems manufactured by a company
 called BioSemi. It can also be read in using :func:`mne.io.read_raw_edf`.
 
+BioSemi amplifiers do not perform "common mode noise rejection" automatically.
+The signals in the EEG file are the voltages between each electrode and CMS
+active electrode, which still contain some CM noise (50 Hz, ADC reference noise,
+etc., see `here <https://www.biosemi.com/faq/cms&drl.htm>`_ for further detail).
```

Comment:
For CircleCI, scroll back in the output to see the warning (which we treat as an error):
```
/home/circleci/project/doc/manual/io.rst:9: WARNING: Duplicate explicit target name: "here".
```
Changing to an anonymous ref ending with `__` might fix it. Otherwise just make it more explicit (and still anonymous) wiht something like:
```
etc., see `the BioSemi FAQ <...>`__
for further details.
```
Might be an improvement anyway.

Reply:
Ah, ok, I see, the ref "here" was set somewhere else alredy right? Thanks, I fixed the ref